code/modules/food/recipes_microwave.dm

```diff
@@ -2797,3 +2797,59 @@
 		/obj/item/weapon/paper,
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/es_cargo
+
+/datum/recipe/steamed_lobster_deluxe //the deluxe comes with butter and a slice of lemon
+	reagents = list (WATER = 10)
+	items = list(
+		/obj/item/weapon/lobster,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/lemon,
+		/obj/item/weapon/reagent_containers/food/snacks/butter,
+		)
+	result = /obj/item/weapon/steamed_deluxe_lobster //this is the lobster whose shell isnt cracked, break its seal and steal its soul, you vile cretin
```

Comment:
I don't know that making an entirely separate object makes sense on this one. Is there any good reason not to just have the food item and give it a var that prevents it from being eaten until wirecuttered?

Reply:
I attempted to create that exact system to massive failure, the var acted way to weird and every fix would cause yet more problems, the best way I found was to create the inedible lobster "weapon" that reacts when wirecuttered